curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md

```diff
@@ -0,0 +1,198 @@
+---
+id: 668f08ea07b99b1f4a91acab
+title: Build a Recipe Page
+challengeType: 14
+dashedName: build-a-recipe-page
+---
+
+# --description--
+
+**Objective:** Build an app that is functionally similar to <a href="https://mot01.github.io/html-lab-1/" target="_blank" rel="noopener noreferrer nofollow">https://recipe-page.freecodecamp.rocks</a>. Do not copy the demo project, give it your own personal style. Fulfill the user stories below and get all the tests to pass to complete the lab. 
```

Comment:
we should update the link to the app so we don't forget to do it later

Reply:
If we are going to have preview pages instead of the demo links, them we can remove the first two sentences from this objective all together

Reply:
I just removed the link for now while we see if we can figure out how to use the preview modal.